### PR TITLE
BZ2151324: Updated namespace requirement for live migration

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.11+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2151324
[CNV-23200](https://issues.redhat.com//browse/CNV-23200)

Link to docs preview: https://55168--docspreview.netlify.app/openshift-enterprise/latest/virt/live_migration/virt-migrating-vm-on-secondary-network.html#virt-configuring-secondary-network-vm-live-migration_virt-migrating-vm-on-secondary-network

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by yossisegev — CHANGES_REQUESTED** [review]

File: modules/virt-configuring-secondary-network-vm-live-migration.adoc [hunk 1]
```diff
@@ -42,9 +42,10 @@ spec:
   }'
 ----
 <1> The name of the `NetworkAttachmentDefinition` object.
-<2> The name of the NIC to be used for live migration.
-<3> The name of the CNI plugin that provides the network for this network attachment definition.
-<4> The IP address range for the secondary network. This range must not have any overlap with the IP addresses of the main network.
+<2> The namespace where the `NetworkAttachmentDefinition` object resides. This must be `openshift-cnv`.
+<3> The name of the NIC to be used for live migration.
+<4> The name of the CNI plugin that provides the network for this network attachment definition.
```
[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply modules/virt-configuring-secondary-network-vm-live-migration.adoc:47]: Thanks for catching this, fixing it now

**Review by yossisegev — COMMENTED** [review]
/LGTM

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**